test/cases/server/plugins/engine/conditions/checkboxes.test.js

```diff
@@ -0,0 +1,78 @@
+import { dirname } from 'node:path'
+import { fileURLToPath } from 'node:url'
+import createServer from '../../../../../../src/server/index.js'
+import FormData from 'form-data'
+
+const testDir = dirname(fileURLToPath(import.meta.url))
+const okStatusCode = 200
+const redirectStatusCode = 302
+const htmlContentType = 'text/html'
+const key = 'wqJmSf'
+
+describe('Checkboxes based conditions', () => {
+  let server
```

Comment:
Not a blocker, but these new tests are great and deserve to be ESLint warning-free 🙌

```suggestion
  /** @type {import('@hapi/hapi').Server} */
  let server
```

Reply:
[Resolved](https://github.com/DEFRA/forms-runner/pull/65/commits/355a74194b0e753f0a5155b16dbfff5dc3ab41a6)